codegen/src/test/resources/software/amazon/awssdk/codegen/poet/waiters/query-async-waiter-interface.java

```diff
@@ -0,0 +1,68 @@
+package software.amazon.awssdk.services;
```

Comment:
Is this package name correct? Seems like it should be under `software.amazon.awssdk.services.myservice.waiters` or something similar

Reply:
Yeah, good call!